ui/src/views/Mailboxes.vue

```diff
@@ -781,7 +781,7 @@
                   for="textInput-modal-markup"
                 >{{$t('mailboxes.name')}}</label>
                 <div class="col-sm-9">
-                  <input type="text" v-model="currentPublic.name" class="form-control">
+                  <input :readonly="currentPublic.isEdit" :disabled="currentPublic.isEdit" type="text" v-model="currentPublic.name" class="form-control">
```

Comment:
we can use `:disabled` instead of `:readonly`, as in all others modules

Reply:
Could you suggest the correct code?